cmd/archeio/main_test.go

```diff
@@ -54,15 +60,22 @@ func TestIntegrationMain(t *testing.T) {
 	testPort := "61337"
 	testAddr := "localhost:" + testPort
 	serverErrChan := make(chan error)
-	cmdContext, serverCancel := context.WithCancel(context.TODO())
-	serverCmd := exec.CommandContext(cmdContext, "archeio")
+	serverCmd := exec.Command("./archeio", "-v=9")
+	serverCmd.Dir = filepath.Join(rootDir, "bin")
 	serverCmd.Env = append(serverCmd.Env, "PORT="+testPort)
-	// serverCmd.Stderr = os.Stderr
-	defer serverCancel()
+	serverCmd.Stderr = os.Stderr
```

Comment:
If you run this with `go test -v ./cmd/archeio` you will get a lot of useful logs for debugging and for confirming the server behavior